can/io/log.py

```diff
@@ -0,0 +1,95 @@
+from can.listener import Listener
+import datetime
+import time
+from can.message import Message
+
+CAN_MSG_EXT = 0x80000000
+CAN_ERR_FLAG = 0x20000000
+CAN_ERR_BUSERROR = 0x00000080
+CAN_ERR_DLC = 8
+
+class canutilsLogReader(object):
+    """
+    Iterator of CAN messages from a .log Logging File (candump -L).
+
+    Only CAN messages are supported (no Errorframes).
+    """
+
+    def __init__(self, filename):
+        self.fp = open(filename, "r")
+
+    def __iter__(self):
+        for line in self.fp:
+            temp = line.strip()
+            if len(temp) > 0:
+                (timestamp, bus, frame) = temp.split()
```

Comment:
This is much nicer that the `.asc` version knowing the names! 
You can remove the unnessasary quotes on the left hand side though.

Reply:
I changed .asc to work with named vars also.